index.js

```diff
@@ -26,10 +24,10 @@ if (fs.existsSync(`${viewDir}/helpers`)) {
 
 const asset = function (file) {
   if (mapping && mapping[file]) {
-    return static(`/_dist/${mapping[file]}`);
+    return staticAsset(`/_dist/${mapping[file]}`);
   }
 
-  return static(file);
+  return staticAsset(file);
```

Comment:
Can we add staticAsset as a global as 'static'? I have a branch that does it, might be easier to just add it in this pr.

Reply:
'static' is a reserved word so re-defining it violates our lint, do we really need to use that?

Reply:
Oh good point, let’s use what you had